### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by sonal-bansal** [twtygqyy/pytorch-SRResNet#23]
Hi ,
I'm training srresnet on DIV2K from scratch(no pretrained) at lr: e-4 , and it is decayed by 0.1 for step=200,
but I'm having MSE loss(at 402 epoch) as:-

Epoch=402, lr=1.0000000000000002e-06
Epoch [402/500], Step [1/50], Loss: 74.55887,Total_loss_old:74.55887
Epoch [402/500], Step [2/50], Loss : 77.55219,Total_loss_old:152.11106
Epoch [402/500], Step [3/50], Loss: 58.70598,Total_loss_old:210.81704
Epoch [402/500], Step [4/50],Loss: 78.69186,Total_loss_old:289.50890
Epoch [402/500], Step [5/50], Loss: 87.79494,Total_loss_old:377.30384
Epoch [402/500], Step [6/50], Loss: 24.87956,Total_loss_old:402.18340
.....
Overall Total_loss: 72.45043201446533

I dont know if actually training or not.
Please check code once:-
[code.zip](https://github.com/twtygqyy/pytorch-SRResNet/files/2828838/code.zip)
This is my result:-
[result_images.docx](https://github.com/twtygqyy/pytorch-SRResNet/files/2828840/result_images.docx)
